server/chat-commands/admin.ts

```diff
@@ -417,14 +417,17 @@ export const commands: ChatCommands = {
 				void IPTools.loadHostsAndRanges();
 				this.sendReply("IPTools has been hot-patched.");
 			} else if (target === 'modlog') {
+				if (!Config.usesqlitemodlog) {
```

Comment:
I would call this something like `usesqlite` because 1) other things will be using SQLite and 2) using the same variable as our temporary one breaks the temporarily useful ability to turn off SQLite logging before doing a migration